applications/dashboard/controllers/api/UsersApiController.php

```diff
@@ -512,32 +485,15 @@ public function index(array $query) {
      * @return array Return a Schema record.
      */
     protected function normalizeOutput(array $dbRecord) {
-        if (array_key_exists('UserID', $dbRecord)) {
-            $userID = $dbRecord['UserID'];
-            $roles = $this->userModel->getRoles($userID, false)->resultArray();
-            $dbRecord['roles'] = $roles;
-        }
-        if (array_key_exists('Photo', $dbRecord)) {
-            $photo = userPhotoUrl($dbRecord);
-            $dbRecord['Photo'] = $photo;
-            $dbRecord['PhotoUrl'] = $photo;
-        }
-        if (array_key_exists('Verified', $dbRecord)) {
-            $dbRecord['bypassSpam'] = $dbRecord['Verified'];
-            unset($dbRecord['Verified']);
-        }
-        if (array_key_exists('Confirmed', $dbRecord)) {
-            $dbRecord['emailConfirmed'] = $dbRecord['Confirmed'];
-            unset($dbRecord['Confirmed']);
-        }
-        if (array_key_exists('Admin', $dbRecord)) {
-            // The site creator is 1, System is 2.
-            $dbRecord['isAdmin'] = in_array($dbRecord['Admin'], [1, 2]);
-            unset($dbRecord['Admin']);
-        }
-
-        $schemaRecord = ApiUtils::convertOutputKeys($dbRecord);
-        return $schemaRecord;
+        $normalizedRow = $this->userModel->normalizeRow($dbRecord, []);
+        $options = [];
+        $result = $this->getEventManager()->fireFilter(
+            'usersApiController_normalizeOutput',
+            $normalizedRow,
+            $this,
+            $options
+        );
```

Comment:
It doesn't look like we had this before. Let's not add it as part of this update.

```suggestion
```